scripts/jenkins/groovy/hadoopCommands.groovy

```diff
@@ -4,7 +4,10 @@
  * @param stageConfig stage configuration to read mode and additional information from
  * @return the cmd used to start H2O in given mode
  */
-def call(final stageConfig) {
+def call(final stageConfig, final boolean getMakeTarget = false) {
+    if (getMakeTarget) {
+        return ""
```

Comment:
q: what's the purpose of this please?

Reply:
for some cases we want to execute another make target (to check that token refresh happended succesfully)

Reply:
https://github.com/h2oai/h2o-3/pull/4551/files#diff-922fd441e2d3ead9c35f17e132b6daecR46